docs/ecosystem.md

```diff
@@ -5,22 +5,30 @@ This document tracks the crates that already exist in the ecosystem that cover f
+|                         | [rust_icu_intl][1]                                             | filmil, kpozin      | Yes              | Uncertain  |                                                                    |
+| `icu::BreakIterator`    | [unicode-segmentation](https://docs.rs/unicode-segmentation/)  | Manish / unicode-rs | No               | No Action  | No line segmentation. Wait for clear user demand outside Rust      |
+| `icu::Collator`         | -                                                              |                     |                  |            |                                                                    |
+| `icu::NumberFormatter`  | [Early Google POC](https://github.com/sffc/rust-wasm-i18n)     | Shane               | ?                | Import     |                                                                    |
+| `icu::CharConversion`   | -                                                              |                     |                  |            |                                                                    |
+| `icu::Char`             | -                                                              |                     |                  |            |                                                                    |
+|                         | [rust_icu_ustring](https://crates.io/crates/rust_icu_ustring)  | filmil, kpozin      | Yes              | Uncertain  |                                                                    |
+| `icu::TimeZone`         | -                                                              |                     |                  |            |                                                                    |
+| `icu::Regex`            | [regex](https://docs.rs/regex/1.3.7/regex/)                    | Core Rust Team      | ?                | No Action  | Main issue: [#37](https://github.com/unicode-org/omnicu/issues/37) |
+| `icu::Calendar`         | [rust_icu_ucal](https://crates.io/crates/rust_icu_ucal)        | filmil, kpozin      | Yes              | Uncertain  | Rust wrapper around ICU4C                                          |
+| `icu::ListFormatter`    | -                                                              |                     |                  |            |                                                                    |
+| `icu::RelativeDateTime` | -                                                              |                     |                  |            |                                                                    |
+| `icu::MessageFormat     | [rust_icu_umsg][2]                                             | filmil, kpozin      | Yes              | Uncertain  |                                                                    |
```

Comment:
Missing backtick

Reply:
Very typical of me to make stupid mistakes like this. :(  Fixed.